codevalidator-api/src/main/java/org/sitenv/vocabularies/loader/code/icd10/Icd10CmLoader.java

```diff
@@ -26,12 +31,7 @@
 	static {
 		CodeLoaderManager.getInstance()
 				.registerLoader(VocabularyConstants.ICD10CM_CODE_NAME, Icd10CmLoader.class);
-		System.out.println("Loaded: " + VocabularyConstants.ICD10CM_CODE_NAME + "(" + VocabularyConstants.ICD10CM_CODE_SYSTEM + ")");
-		
-		if (VocabularyRepository.getInstance().getVocabularyMap() == null)
```

Comment:
* These blocks (as were found in the static initializers of all the loaders) create a potential, but unlikely, race condition if 2x loader classes were in the midst of class loading (the vocabulary map could be reset).
* Fixed by simply making the map instance initialized on the VocabularyRepository instance.